ChangeLog

```diff
@@ -1,5 +1,18 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x released xxxx-xx-xx
+
+Changes
+   * Added config.h option MBEDTLS_NO_UDBL_DIVISION, to prevent the use of
+     64-bit division.
+   * Added config.h option MBEDTLS_TYPE_UDBL to allow configuring the
```

Comment:
@hanno-arm No! The point of `MBEDTLS_NO_UDBL_DIVISION` is specifically to indicate the availability of double-width integer division. There are many environments where a double-width integer type exists, and using it for “simple” operations such as bit masks is fine because the compiler does the job. But division on this type requires a software implementation, which needs to be linked into the program, and in some environments linking the correct library is not convenient. See issue #708 .